### ISSUE TITLE
Cannot remove from server

### ISSUE THREAD

**Issue by TheMightyMcGrew** [ForgeEssentials/ForgeEssentials#2455]
I've installed this once in the past, but removed it later because I thought it was causing issues. Turns out it wasnt and those issues were fixed and was going to go back to using it but the previous settings were still in affect even after removing all the files and folder related, and to make matters worse the removed files screwed up the backup system and started a backup every 5-10 seconds of about 6-7 dimensions which caused the server file to grow from 8gb to 20gb in under 10 minutes. How can i completely reset everything related to FE without doing a complete server overhaul?

**Comment by emilyastranova** [comment]
Does the same thing happen using a different save? If so it's definitely a config issue.

**Comment by spacebuilder2020** [comment]
Yea, that is definitely weird.  Default config for backups is only once every hour or on world load/unload.

Perhaps some process is triggering the backup by loading and unloading the worlds.  Try disabling backup on world load/unload in the configs.

Also, if you don't need the backup module, you can always disable it.

**Comment by Zethalion** [comment]
@TheMightyMcGrew is this issue resolved for you?

**Comment by llazzllo** [comment]
I'm having a similar issue. I installed Forge Essentials and decided to remove it. I deleted the mod, and the folder it created(I didn't find a config), but I can't get Corpse Complex to work the way I had it set up.  I've replaced the config for corpse complex,  tried restarting the server without it and then again with it added back in, and nothing has worked. Forge Essentials changed something that did not change back once it was removed.
